DragonFruit.Six.API/Stats/SeasonalStats.cs

```diff
@@ -39,6 +48,31 @@ public class Season
         [JsonProperty("maxmmr")]
         public double MaxMMR { get; set; }
 
+        [JsonProperty("next_rank_mmr")]
+        public double NextRankMMR { get; set; }
+
+        [JsonProperty("previous_rank_mmr")]
+        public double PreviousRankMMR { get; set; }
+
+        [JsonProperty("skill_mean")]
+        public double SkillMean { get; set; }
+
+        [JsonProperty("skill_stdev")]
+        public double SkillStdev { get; set; } // TODO: Wasn't sure what to call this... Needs to be named more appropriately
+
+        [JsonProperty("last_match_result")]
+        public uint LastMatchResult { get; set; }
+
+        [JsonProperty("last_match_mmr_change")]
+        public double LastMatchMMRChange { get; set; }
+
+        [JsonProperty("last_match_skill_mean_change")]
+        public double LastMatchSkillChange { get; set; }
+
+        [JsonProperty("last_match_skill_stdev_change")]
+        public double LastMatchSkillStdevChange { get; set; } // TODO: Wasn't sure what to call this... Needs to be named more appropriately
+
```

Comment:
Add `top_rank_position`? It equals `0` on a user that isn't in the top 10k so a `uint` will suffice here

Reply:
I have added this in my next commit.